driver/src/main/java/oracle/nosql/driver/NoSQLHandle.java

```diff
@@ -571,6 +571,15 @@ public SystemResult doSystemRequest(String statement,
                                         int timeoutMs,
                                         int pollIntervalMs);
 
+    /**
+     * Returns configuration object for client side statistics.
```

Comment:
"client side" seems to be redundant in user-facing material. It's useful when we talk about it internally but for an SDK user I think it should be simply "statistics"